src/olympia/addons/models.py

```diff
@@ -479,9 +479,6 @@ def delete(self, msg='', reason=''):
         for preview in previews:
             tasks.delete_preview_files.delay(preview)
 
-        # Remove from search index.
-        tasks.unindex_addons.delay([id])
```

Comment:
Since the code a few lines above just does a `self.update()` we just need to remove this in order for the deleted add-ons to be properly indexed. In fact before both tasks were triggered, which could lead to unexpected results...